Avoid wrapping in hinted `opam install` commands

CraigFe opened the pull request:
This avoids needing to manually undo the wrapping when copying the suggested command into a terminal.

I also added a forced newline to ensure that the suggested command is always on its own line. This is somewhat questionable, but in my terminal, at least, this allows selecting the entire command with a double-click rather than a click and drag. If this is deemed worthwhile, I can make similar changes to the other hinted commands.

jeremiedimino commented:
Forced newline seems fine, maybe with some indentation?

As discussed on slack, having a function for such command hints would be great!

CraigFe commented:
Apologies for letting this PR stagnate.

I've updated it to cover all instances of `try: ...` in the codebase (using a pretty-printer defined in `src/dune/utils.ml` for lack of a better place), and added 2 spaces of indentation to the command.

jeremiedimino commented:
`Utils` will do. If the number of pp utilities grows, we could have a dedicated module for them.
